docs/includes/generated_docs/schemas/beta.core.md

```diff
@@ -0,0 +1,357 @@
+# <strong>beta.core</strong> schema
+
+Available on backends: [**TPP**](../../backends#tpp), [**EMIS**](../../backends#emis)
+
+This schema defines the core tables and columns which should be available in any backend
+providing primary care data, allowing dataset definitions written using this schema to
+run across multiple backends.
+
+!!! warning
+    This schema is very much still a work-in-progress while the EMIS backed remains
+    under developemnt.
+
+``` {.python .copy title='To use this schema in an ehrQL file:'}
+from ehrql.tables.beta.core import (
+    clinical_events,
+    medications,
+    ons_deaths,
+    patients,
+)
+```
+
+<p class="dimension-indicator"><code>many rows per patient</code></p>
+## clinical_events
+
+
+
+<dl markdown="block" class="schema-column-list">
+<div markdown="block">
+  <dt id="clinical_events.date">
+    <strong>date</strong>
+    <a class="headerlink" href="#clinical_events.date" title="Permanent link">🔗</a>
+    <code>date</code>
+  </dt>
+  <dd markdown="block">
```

Comment:
thought: This HTML is needed throughout because Python's Markdown support isn't flexible to write it in Markdown. So it's slightly clunky and repetitive, but it's fine: we're not hand-editing this form of the content. At worst, it would be the template that's edited.

There are:

* [definition lists](https://python-markdown.github.io/extensions/definition_lists/)
* [attribute lists](https://python-markdown.github.io/extensions/attr_list/), but these don't support code blocks, so don't know if they would even work

Reply:
Yes, I went through exactly this thought process myself. Initially it felt like a hack, but I think for auto-generated stuff like this it's fine to resort to raw HTML to get the layout or structure we want because it's easy to change later.